loop/payment_template.py

```diff
@@ -199,26 +199,23 @@ def excel_processing(workbook, name_of_sheets, heading_of_sheets, heading_format
                                     cell_value_from_headers.get('formula_list'),
                                     row_format)
 
-            
     except Exception as e:
         print e
     return workbook
 
 
 def prepare_value_data_generic(data):
-    data = json.loads(data)
+    if type(data) == 'str':
```

Comment:
What if type(data) is not 'str' then should we proceed to remaining code or terminate here with error message ?